src/pygeoprocessing/routing/routing.pyx

```diff
@@ -2768,14 +2814,17 @@ def distance_to_channel_mfd(
                     # nodata flow, so we skip
                     continue
 
+                # the algorithm starts from an arbitrary pixel
+                # it pushes its downstream neighbors to a stack, working depth-first
+                # until reaching either a stream or a raster edge
                 if visited_managed_raster.get(xi_root, yi_root) == 0:
                     visited_managed_raster.set(xi_root, yi_root, 1)
                     # arguments are x,y position of pixel, then last D8 flow
                     # direction processed (0-7), and last is the running
                     # accumulation distance accumulated by this pixel so far
                     # initialized to nodata

```

Comment:
These comments look to be out of date for this function now, given that we're using MFD and a different struct.

Reply:
removed this comment block